src/main/java/duke/AddCommand.java

```diff
@@ -0,0 +1,58 @@
+package duke;
+
+public class AddCommand implements Command {
+
+    private String commandWord;
+    private String description;
+    static String defaultString = "Got it. I've added this task:";
+
+    AddCommand(String commandWord, String description) {
+        this.commandWord = commandWord;
+        this.description = description;
+    }
+
+    public void execute(TaskList tasks, Ui ui) throws DukeException {
+        Task task;
+        String[] stringSplit;
```

Comment:
for naming convention sake maybe you want to consider changing it to something like stringComponents?